goalgorilla/open_social
# Issue #3239820 by nkoporec: Landing page visibility label alignement issues when using the new sky styling

## Problem
When using the new sky styling, the landing page visibility label is not aligned with the rest of the types (basic page, event ...) when displayed in a teaser.

## Solution
We need to add a new template override for node--landing-page--featured-sky, that will extend the node--featured-sky template.

## Issue tracker
https://www.drupal.org/project/social/issues/3239820

## How to test
1. Enable the new sky style
2. Enable social_landing_page
3. Create a landing page and add a section (Add featured content) with some landing page, events and topic nodes
4. See the visibility label

## Screenshots
Before:
![2021-09-29_11-44](https://user-images.githubusercontent.com/35064680/135246878-6db65077-ca84-4629-b100-e71d595f470d.png)

After:
![2021-09-29_11-58](https://user-images.githubusercontent.com/35064680/135247151-cea0f463-7d7e-4eea-baff-e1411ea32dfc.png)


## Release notes
We've fixed an issue where landing page visibility label was not aligned with other content types when displayed in a teaser.

## Change Record

## Translations
